applications/blackhole/doc/index.md

```diff
@@ -32,37 +34,90 @@ From here, you can write your own Javascript callbacks, triggered everytime a re
-      socket.emit('subscribe', { account_id: '{ACCOUNT_ID}', auth_token: '{AUTH_TOKEN}', binding: 'call.CHANNEL_DESTROY.*' });
-      socket.emit('subscribe', { account_id: '{ACCOUNT_ID}', auth_token: '{AUTH_TOKEN}', binding: 'conference.event.*' });
-
-      socket.on('participants_event', function (data) {
-        console.log(data);
-      });
-      socket.on('CHANNEL_CREATE', function (EventJObj) {
-        console.log(EventJObj);
-      });
-      socket.on('CHANNEL_ANSWER', function (EventJObj) {
-        console.log(EventJObj);
-      });
-      socket.on('CHANNEL_DESTROY', function (EventJObj) {
-        console.log(EventJObj);
-      });
-
-
-      var Events = ['connect', 'error', 'disconnect', 'reconnect', 'reconnect_attempt', 'reconnecting', 'reconnect_error', 'reconnect_failed'];
-      for (idx in Events) {
-        socket.on(Events[idx], function (_evt) { console.log('=ERROR REPORT==== ' + Events[idx]); });
-      }
+        var socket = io.connect('http://{BLACKHOLE_IP_ADDRESS}:5555');
+
+        socket.emit('subscribe', {
+            account_id: '{ACCOUNT_ID}',
+            auth_token: '{AUTH_TOKEN}',
+            binding: 'call.CHANNEL_CREATE.*'
+        });
+
+        socket.emit('subscribe', {
+            account_id: '{ACCOUNT_ID}',
+            auth_token: '{AUTH_TOKEN}',
+            bindings: ['call.CHANNEL_ANSWER.*', 'call.CHANNEL_DESTROY.*']
+        });
+
+        socket.emit('subscribe', {
+            account_id: accountId,
+            auth_token: token,
+            bindings: ['doc_created.*.user.*', 'doc_edited.*.user.*']
+        });
+
+        socket.on('CHANNEL_CREATE', function(EventJObj) {
+            console.log(EventJObj);
+        });
+
+        socket.on('CHANNEL_ANSWER', function(EventJObj) {
+            console.log(EventJObj);
+        });
+
+        socket.on('CHANNEL_DESTROY', function(EventJObj) {
+            console.log(EventJObj);
+        });
+
+        socket.on('doc_created_user', function(EventJObj) {
+            console.log(EventJObj);
+        });
+
+        socket.on('doc_edited_user', function(EventJObj) {
+            console.log(EventJObj);
+        });
+
+        // socket.emit('unsubscribe', {
+        //     account_id: accountId,
+        //     auth_token: token,
+        //     bindings: ['doc_created.*.user.*', 'doc_edited.*.user.*']
+        // });
+
+        // socket.emit('unsubscribe_all', {
+        //     account_id: accountId,
+        //     auth_token: token
+        // });
+
+        var Events = ['connect', 'error', 'disconnect', 'reconnect', 'reconnect_attempt', 'reconnecting', 'reconnect_error', 'reconnect_failed'];
+
+        for (var idx in Events) {
+            socket.on(Events[idx], function(_evt) {
+                console.log('=ERROR REPORT==== ' + Events[idx]);
+            });
+        }
     </script>
   </body>
 </html>
 ```
-You can also add a friendly name and some metadata to any subscribe command.
 
+You can add one or multiple bindings by using:
```

Comment:
You should add a note that `binding` is taken before `bindings` in case someone sets both.